.github/workflows/test.yaml

```diff
@@ -31,25 +44,34 @@ jobs:
         ROLLBAR_ACCESS_TOKEN: ${{ secrets.ROLLBAR_ACCESS_TOKEN }}
         ROLLBAR_USERNAME: ${{ github.actor }}
 
+    - uses: actions/download-artifact@v2
+      with:
+        name: bundle1.js.map
+    - uses: actions/download-artifact@v2
+      with:
+        name: bundle2.js.map    
+
     # Print Rollbar's deploy id
     - name: Get the Rollbar deploy id
       run: echo Deploy created with id ${{ steps.rollbar_pre_deploy.outputs.deploy_id }}
 
     - name: Deploy
       run: echo Deploying...
 
-    - name: Nofity rollbar deploy succeeded
+    - name: Notify rollbar deploy succeeded
       uses: ./
       id: rollbar_post_deploy
       with:
         environment: 'production'
         version: ${{ github.sha }}
         status: 'succeeded'
+        source_map: bundle1.js.map bundle2.js.map
```

Comment:
@mrunalk i think we made a typo here. These properties are plural `source_maps` and `minified_urls`.

Reply:
Ahh...thanks for catching that. I wonder why it wasn't picked up the test itself then? Not sure if our test reliable then.